src/analyzer.ts

```diff
@@ -117,6 +120,15 @@ const emitDefinitelyUsed = (file: SourceFile, onResult: OnResultType) => {
 const emitPotentiallyUnused = (file: SourceFile, onResult: OnResultType) => {
   const exported = getExported(file);
 
+  const idsInFile = file.getDescendantsOfKind(ts.SyntaxKind.Identifier);
+  const referencedInFile = _(idsInFile)
```

Comment:
Would you kindly refactor this to not use `_` chaining please? You're free to use the `fp` constructs as you wish :)

Somethings like:

```ts
const referenceCounts = countBy((idsInFile || []).map(node => node.getText()));
const referencedInFile = Object.entries(referenceCounts)...
```

Reply:
Sure, done, though I am curious what the objection to chaining is. It has to be `countBy(x => x)(...)`, the iteratee seems to be required w/ lodash-fp.

Reply:
You can write it alternatively as `countBy(x => x, input)` as well.

It's more of a preference to use the `fp` methods, than a hesitation to use chaining. I find the `fp` methods work on immutable data, have consistent interfaces, and can be chained with `pipe`.